src/components/pages/Cases/CaseTable.js

```diff
@@ -536,6 +483,20 @@ export default function CaseTable(props) {
               />
             </div>
           </TabPane>
+          <TabPane
+            tab={
+              <Button
+                className="save-cases-btn"
+                onClick={() => {
+                  bookmarkCases(selectedRowID);
+                }}
+              >
+                <FeatherIcon icon="bookmark" />
+              </Button>
+            }
+            disabled
+            key="3"
+          />
```

Comment:
It looks like you are putting the save button in another tab, I would like to see how this work when you select the cases to be saved. For styling, this could work better. I think we could have kept the save button that as there previously instead of going back to the feathericon